.github/workflows/test.yml

```diff
@@ -41,13 +41,40 @@ jobs:
         with:
           command: fmt
           args: --all -- --check
+      - name: Run cargo test with blocking client
+        if: matrix.gremlin-server == '3.5.7'
+        uses: actions-rs/cargo@v1
+        with:
+          command: test
+          args: --manifest-path gremlin-client/Cargo.toml
```

Comment:
By chance I bumped into a compilation error for the blocking I/O side fixed in d56179dc235874a49ab8ce0be79b59db4e9e84b3 while working on a follow-up branch for this PR. Surprised me since this PR had been building successfully, but that's when I noticed there didn't seem to be a cargo test call without either async flag. If there was the build should have broken, so I added one.